packages/manager/modules/iam/src/dashboard/dashboard.template.html

```diff
@@ -9,12 +9,19 @@
                     data-translate="iam_dashboard_header_tabs_item_policies"
                 ></span
             ></oui-header-tabs-item>
-            <oui-header-tabs-item
-                data-state="iam.dashboard.identitiesRedirection"
-                ><span
-                    data-translate="iam_dashboard_header_tabs_item_identities"
-                ></span
-            ></oui-header-tabs-item>
+            <div class="oui-header-tabs-item">
+                <div class="oui-header-tabs__item">
+                    <a
+                        class="oui-header-tabs__action"
+                        target="_top"
+                        data-ng-href="{{ $ctrl.accountUsersURL }}"
+                    >
+                        <span
+                            data-translate="iam_dashboard_header_tabs_item_identities"
+                        ></span>
+                    </a>
+                </div>
+            </div>
```

Comment:
I used the native version of the tab because we can't pass the `target="_top"` property to the component version, required by the container